src/event-notion/index.ts

```diff
@@ -234,9 +234,13 @@ export const syncHostFormToNotionCalendar = async (config: EventNotionPipelineCo
       });
       newEventRows.splice(index, 1);
       // Report error in Discord as well.
+      let errorMsg = error.message;
+      if (error instanceof RangeError) {
+        errorMsg = "The date couldn't be parsed correctly. Make sure the start time is later than the end time!";
```

Comment:
Can we attach the message when we originally throw the RangeError in NotionCalEvent.ts? Then here we can just use error.message, and we don't even need another if check for the instance of RangeError.